x-pack/plugins/observability_solution/infra/public/alerting/log_threshold/components/alert_details_app_section/components/logs_history_chart.tsx

```diff
@@ -42,6 +51,7 @@ const LogsHistoryChart = ({ rule }: { rule: Rule<PartialRuleParams> }) => {
   const executionTimeRange = {
     gte: DateMath.parse(dateRange.from)!.valueOf(),
     lte: DateMath.parse(dateRange.to, { roundUp: true })!.valueOf(),
+    buckets: 30,
```

Comment:
`NUM_BUCKETS` is 20 and I think it was added for the preview chart, not the alert history chart.

When I was checking the history chart, despite the fact that I had data for a couple of hours, I was seeing 2 bars that were confusing because we wanted to have a history chart for the last 30 days and one bar per day which was not the case. So, I added this bucket size to control the exact number of buckets that are shown on the chart.

Please let me know if my assumption is not correct.